prometheus/exporter/gpu_exporter.py

```diff
@@ -42,17 +42,17 @@ def parseSmiXmlResult(smi, logDir):
         outPut[str(minorNumber)] = {"gpuUtil": gpuUtil, "gpuMemUtil": gpuMemUtil}
     return outPut
 
-def genGpuMetricsFromSmi(logDir): 
+def genGpuMetricsFromSmi(logDir):
     try:
         cmd = "nvidia-smi -q -x"
         smi_output = subprocess.check_output([cmd], shell=True)
         return parseSmiXmlResult(smi_output, logDir)
     except subprocess.CalledProcessError as e:
-        raise RuntimeError("command '{}' return with error (code {}): {}".format(e.cmd, e.returncode, e.output))
+        print("command '{}' return with error (code {}): {}".format(e.cmd, e.returncode, e.output))
```

Comment:
>print [](start = 8, length = 5)

1. not familiar with Python, do you need to explicitly return null here?
2. can you capture only exception caused by command not found and raise other exception?

Reply:
for
1 current method return value will be null when exception trigger. no need to explicitly return null.
2 refine at next commit